parameter-sets/oauth-credentials/parameter-set.json

```diff
@@ -4,8 +4,8 @@
         "description": "",
         "icon": "icon-calendar"
     },
-    "defaultDefinableInline": true,
-    "defaultDefinableAtProjectLevel": true,
+    "defaultDefinableInline": false,
+    "defaultDefinableAtProjectLevel": false,
```

Comment:
Why did these need to be changed? They are still true for the googledrive plugin which has a similar fix. 
(I'm not sure exactly what they do)

Reply:
OK sorry about that, I should have changed them in googledrive too 😢 
They rule the default settings when the DSS admin creates a new preset (which means they can still override it).
These two settings allow
- the end user to defined a "Manually defined" in the dataset / recipe itself. Here, since the preset is only about OAuth2 SSO, it does not make sense (because the core DSS is needed to go through the whole SSO dance)
- the preset can be defined and used at project level. That would make sense, but currently it's not fully working.

Reply:
Ok I see them now in the preset form - that all makes sense - thanks!